Eevee.Sleep.Bot/Models/InGameAnnouncement/InGameAnnouncementDetailModel.cs

```diff
@@ -0,0 +1,16 @@
+using JetBrains.Annotations;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Eevee.Sleep.Bot.Models.InGameAnnouncement;
+
+[BsonIgnoreExtraElements]
+public record InGameAnnouncementDetailModel: InGameAnnouncementMetaModel {
+    [UsedImplicitly]
+    public required string Content { get; init; }
+
+    [UsedImplicitly]
+    public required string ContentHash { get; init; }
+
+    [UsedImplicitly]
+    public required string Updated { get; init; }
```

Comment:
Store type of `Date` if it's from the actual announcement

Reply:
It is given in the format yyyy/M/dd, which makes it over-represented in Datetime (and annoying to parse).
DateOnly requires conversion to Datetime when saving to MongoDB
So I thought it would be better to save the data as a string.

Reply:
Also, if Updated is currently stored as a string, migration is required.
Or delete all detail/history records

Reply:
Noice
I would just delete all records and let it re-scrape, it's not live anyway.